plugins/woocommerce/templates/global/quantity-input.php

```diff
@@ -41,7 +41,7 @@
 		class="<?php echo esc_attr( join( ' ', (array) $classes ) ); ?>"
 		name="<?php echo esc_attr( $input_name ); ?>"
 		value="<?php echo esc_attr( $input_value ); ?>"
-		title="<?php echo esc_attr_x( 'Qty', 'Product quantity input tooltip', 'woocommerce' ); ?>"
+		aria-label="<?php echo esc_attr_e( 'Product quantity', 'woocommerce' ); ?>"
```

Comment:
`esc_atrr_e()` already outputs the string, so we don't need the `echo` before. We can either drop the `echo` or use `esc_attr__()` instead.